examples/adapt.ipynb

```diff
@@ -66,14 +66,14 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/alex/Codes/Tangelo/tangelo/algorithms/variational/vqe_solver.py:260: RuntimeWarning: No variational gate found in the circuit.\n",
+      "/Users/krzysztofbieniasz/Code/env/lib/python3.9/site-packages/tangelo/algorithms/variational/vqe_solver.py:260: RuntimeWarning: No variational gate found in the circuit.\n",
```

Comment:
If no changes were applied to this file other than metadata, then I have nothing to say.

Reply:
Yeah, maybe I should have just reverted to older version

Reply:
I don't think we care, I say leave it alone.